### ISSUE TITLE
Classes with numbers are invalid

### ISSUE THREAD

**Issue by jquesada2016** [gbj/leptos#73]
The following valid class name does not compile because of the `25`:

```rust
view! { cx,
  <span class-opacity-25=true />
}
```

**Comment by gbj** [comment]
Thanks. This is a limitation of `syn-rsx` at the moment and I'll look into opening a PR over there.

**Comment by gbj** [comment]
By the way, I believe `class-opacity25` *is* valid (i.e., without the dash). The issue is not the numbers per se, that `syn-rsx` parses each dash-separated piece of the attribute name as a `syn::Ident`, and `25` is an invalid Rust ident.

**Comment by gbj** [comment]
So this is fixed by a0a66b75ddbbd9e49bdbf338499a96b8a7e8a1e3, and the newly-available syntax for the original case is
```rust
view! { cx,
  <span class=("opacity-25", true)/>
}
```
